README.md

```diff
@@ -1,4 +1,4 @@
-# LEAD-Center &middot; [![Build Status](https://travis-ci.org/eilenshahbaz/LEAD-Center.svg?branch=master)](https://travis-ci.org/eilenshahbaz/LEAD-Center)
+# LEAD-Center &middot; [![Build Status](https://travis-ci.org/eilenshahbaz/LEAD-Center.svg?branch=master)](https://travis-ci.org/eilenshahbaz/LEAD-Center) &middot; [![Maintainability](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/maintainability)](https://codeclimate.com/github/codeclimate/codeclimate/maintainability) &middot; [![Test Coverage](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/test_coverage)](https://codeclimate.com/github/codeclimate/codeclimate/test_coverage)
```

Comment:
From the [API Docs](https://codeclimate.com/github/codeclimate/codeclimate/badges#test-coverage-markdown), it looks good! Thanks for setting this up 👍